change/@ni-nimble-components-bded907f-ef59-4ed0-915b-07882e96bd9e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is there anything we can do about the action menu getting clipped by the bounds of the table?

This is from the [Angular example app](https://60e89457a987cf003efc0a5b-gfpntrqrjo.chromatic.com/example-client-app/#/customapp) in Firefox on macOS. In Safari and Edge on macOS it doesn't clip.

![image](https://user-images.githubusercontent.com/10500124/218877606-06a12a2e-184b-40e7-a676-e6b415da7bcf.png)

Reply:
I adjusted the styling to resolve this problem.